### ISSUE TITLE
Some questions about libicu when build 2.1.1

### ISSUE THREAD

**Issue by zhegexiaohuozi** [ariya/phantomjs#14101]
use the `build.py` in 2.1.1,I can build sucess,but it is not fully static.I can find some icu libs when use cmd `ldd`.But I download phantomjs(linux x64) from offical website and excute `ldd`,I get  the following:
```
[root@ip-xxx bin]# ldd phantomjs 
        linux-vdso.so.1 =>  (0x00007ffc33f05000)
        libz.so.1 => /lib64/libz.so.1 (0x00007fcf5958e000)
        libfontconfig.so.1 => /usr/lib64/libfontconfig.so.1 (0x00007fcf59359000)
        libfreetype.so.6 => /usr/lib64/libfreetype.so.6 (0x00007fcf590bc000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00007fcf58eb8000)
        librt.so.1 => /lib64/librt.so.1 (0x00007fcf58cb0000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007fcf58a93000)
        libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00007fcf5878e000)
        libm.so.6 => /lib64/libm.so.6 (0x00007fcf5848c000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007fcf58275000)
        libc.so.6 => /lib64/libc.so.6 (0x00007fcf57eb3000)
        /lib64/ld-linux-x86-64.so.2 (0x00005557dc377000)
        libexpat.so.1 => /lib64/libexpat.so.1 (0x00007fcf57c8a000)
```
Can not find icu...,it is very cool,so,how did you do that? Can you also commit the method to `build.py`?
Thank you very much!

**Comment by Vitallium** [comment]
Hi, you need to build static version of libicu.

**Comment by zhegexiaohuozi** [comment]
@Vitallium 
Yeah,I build a static version by the following cmd
```
runConfigureICU Linux --enable-static --disable-shared --with-data-packaging=archive
make && make install
```
(my os: centos 6.6 x64)
the icu will be installed in `/usr/local/include` and `/usr/local/lib`,then I build phantomjs with the following cmd
```
python build.py --confirm --qt-config "-I /usr/local/include/ -L /usr/local/lib/"
```
but I got `The ICU library support cannot be enabled.`,something wrong?

**Comment by ariya** [comment]
We don't add this to `build.py` because this is a special case for our binary. Under normal circumstances, you don't want to do this.

If you want to see how it's done, check [deploy/docker-build.sh](https://github.com/ariya/phantomjs/blob/master/deploy/docker-build.sh). It seems that your static ICU failed or it is conflicting with the existing libraries.

In all cases, this is likely a build problem on your system as it's working just fine for our binaries.

**Comment by zhegexiaohuozi** [comment]
Fixed in merge request [#14148](https://github.com/ariya/phantomjs/pull/14148).

**Comment by ariya** [comment]
Fixed with PR #14148. Thank you!
